lib/timeline/timeline.dart

```diff
@@ -76,18 +71,19 @@ class TimelineScreen extends Screen {
       ..disabled = true
       ..click(_resumeRecording);
 
+    final CoreElement upperButtonSection = div(c: 'section')
+      ..layoutHorizontal()
+      ..add(<CoreElement>[
+        pauseButton,
+        resumeButton,
+        div()..flex(),
+      ]);
+    getServiceExtensionButtons().forEach(upperButtonSection.add);
```

Comment:
This is existing code, but I believe an equally effective way to write this would be:

```
upperButtonSection.add(getServiceExtensionButtons());

```

Reply:
done.